src/orca/infos.ts

```diff
@@ -33,8 +33,15 @@ infos = class InstanceOrca {
 
     const filters = [sizeFilter];
     const config: GetProgramAccountsConfig = { filters: filters };
-    const allOrcaPool = await connection.getProgramAccounts(ORCA_POOL_PROGRAM_ID, config);
-
+    let allOrcaPool = await connection.getProgramAccounts(ORCA_POOL_PROGRAM_ID, config);
+
+    allOrcaPool.sort((a, b) => a.pubkey.toBase58().localeCompare(b.pubkey.toBase58()));
+    if (page) {
```

Comment:
1. Perhaps this piece of logic can be extracted to utils as a function? Ex: `Paginate`
2. Assign the batched accounts to a new variable (Ex: `pagedPoolAccounts`) instead of mutating `allV4AMMAccount`